src/ranking/effectiveness/channel.ts

```diff
@@ -155,14 +155,14 @@ export namespace PreferredAxisScore {
     return score;
   }
 
-  export function featurize(type: ExtendedType, channel: Channel) {
+  export function featurize(type, channel) {
```

Comment:
Not sure why you remove `: ExtendedType` here.  Incorrect merge conflicts resolution?